src/repository/models.py

```diff
@@ -432,7 +427,7 @@ class Preprint(models.Model):
         null=True,
         on_delete=models.SET_NULL,
     )
-    comments_editor = models.TextField(
+    comments_editor = model_utils.JanewayBleachField(
```

Comment:
This hasn't previously been rich text:

<img width="727" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/8321378/1e8fbf43-e77c-40f4-b105-5a8baa70957f">